npm/vite-dev-server/README.md

```diff
@@ -56,10 +56,25 @@ We then merge the sourced config with the user's vite config, and layer on our o
 | <= v2                    | <= v9   |
 | >= v3                    | >= v10  |
 
+#### `devServerPublicPathRoute` for Vite v5
```

Comment:
Seems really weird to me that we don't have specific bundler docs for CT. Shouldn't this be documented as an option in the component config here? https://docs.cypress.io/guides/references/configuration#component

Reply:
I would think so, but I am not sure why it isn't documented. Was it something we wanted as an option but not to be widely surfaced or was the documentation just missed?